README.md

```diff
@@ -36,19 +49,30 @@ The config.inc.php settings format is:
 [ithenticate]
 
 ; Enable iThenticate to submit manuscripts after submit step 4
-;ithenticate = On
+; ithenticate = On
 
 ; Credentials can be set by context : specify journal path
```

Comment:
The examples/configuration here are OJS specific (referring to journals); make it general so it's helpful for OMP and OPS.